src/components/CytoscapeLayout/CytoscapeLayout.tsx

```diff
@@ -25,7 +25,19 @@ export default class CytoscapeLayout extends React.Component<CytoscapeLayoutProp
     };
   }
 
+  resizeWindow() {
+    let canvasWrapper = document.getElementById('cytoscape-container')!;
+
+    if (canvasWrapper != null) {
+      let dimensions = canvasWrapper.getBoundingClientRect();
+      canvasWrapper.style.height = `${document.documentElement.scrollHeight - dimensions.top}px`;
+    }
+  }
+
   componentDidMount() {
+    window.addEventListener('resize', this.resizeWindow);
```

Comment:
Is there any equivalent for umount? What happens when you navigate to other window? Does it "reloads" the page or this event will continue there?